### ISSUE TITLE
Explanation on how it works

### ISSUE THREAD

**Issue by Guptajakala** [pyni/handeye_calibration_with_depth_camera#1]
Hi, thanks for the great repo!
Is there any explanations about how it works (e.g. why is the calibration board this shape, how is depth data used with aruco marker)?

**Comment by pyni** [comment]
hi, you can check this:

**Comment by pyni** [comment]
https://zhuanlan.zhihu.com/p/157299575

**Comment by Guptajakala** [comment]
I saw you included super4pcs, but you didn't actually use it for calibration, right? You are using aruco marker + icp?

**Comment by pyni** [comment]
yes, aruco marker has provided with a good initial pose. Super4pcs usually is designed for other calibration blocks, such as this:

**Comment by pyni** [comment]
![Uploading image.png…]()

**Comment by Guptajakala** [comment]
The image uploading seems to have failed.

When attaching the markers on the 3D printed biaodingkuai.stl, does the marker has to bee attached in some certain positions so that you willl use their relative position w.r.t. the biaodingkuai ?
